Dockerfile

```diff
@@ -0,0 +1,40 @@
+#advancecomp
```

Comment:
Please do not use git clone. Also, these wgets will likely slow down build considerably depending on network bandwidth/availability, is there another way you can seed the corpus (for example, you are allowed to make a testsuite directory as part of your integration).